doc/ems-application-guide/src/energyplus-runtime-language/variables.tex

```diff
@@ -98,14 +98,16 @@ \subsection{Built-In Variables}\label{built-in-variables}
 IsRaining & 0 (= no) or 1 (= yes) \tabularnewline
 ZoneTimeStep & Durating of the zone timestep in hours \tabularnewline
 SystemTimeStep & Current duration of the system timestep in hours \tabularnewline
-CurrentEnvironment & Integer index for the current environment period (sizing periods and run periods accumulate throughout the run) \tabularnewline
+CurrentEnvironment & Integer index for the current environment period (sizing periods and run periods accumulate throughout the run). \tabularnewline
 Null & Special structure that stops an actuator from overriding control \tabularnewline
 ActualDateAndTime & A simple sum of the values of the date/time function. Could be used in random seeding. \tabularnewline
 ActualTime & A simple sum of the values of the time part of the date/time function. Could be used in random seeding. \tabularnewline
 WarmupFlag & 0 (= not during warmup) or 1 (= during warmup) \tabularnewline
 \bottomrule
 \end{longtable}
 
+For the built-in variable CurrentEnvironment, it depends on how many SizingPeriod modules and RunPeriod modules are activated. For example, if SizingPeriod:DesignDay, SizingPeriod:WeatherFileDays, SizingPeriod:WeatherFileConditionType, RunPeriod are all specified, then the index for each one is: 1 for heating design day, 2 for cooling design day, 3 for SizingPeriod:WeatherFileDays, 4 for SizingPeriod:WeatherFileConditionType, and 5 for RunPeriod. For accurate index values, please lookup the table EnvironmentPeriods in eplusout.sql file.
```

Comment:
This is good, but I have some edits to suggest:
"For the built-in variable CurrentEnvironment, it depends on how many SizingPeriod and RunPeriod objects are activated in the idf file. For example, if there are two SizingPeriod:DesignDay objects (winter, then summer), plus one each of SizingPeriod:WeatherFileDays, SizingPeriod:WeatherFileConditionType, and RunPeriod . . . "

and at the end:
". . . in the eplusout.sql file or refer to the order of the Environment reports in the eio output."